File: 3581-the-two-sneaky-numbers-of-digitville/3581-the-two-sneaky-numbers-of-digitville.py
class Solution:
    def getSneakyNumbers(self, nums: List[int]) -> List[int]:
        nums.sort()
        l=[]
        for i in range(len(nums)-1):
            if((nums[i]^nums[i+1])==0):
                l.append(nums[i])
        return l
        

File: 3581-the-two-sneaky-numbers-of-digitville/3581-the-two-sneaky-numbers-of-digitville.cpp
class Solution {
public:
    vector<int> getSneakyNumbers(vector<int>& nums) {
        sort(nums.begin(),nums.end());
        vector<int>ne;
        for (int i=0;i<nums.size()-1;i++){
            if((nums[i]^nums[i+1])==0){
                ne.push_back(nums[i]);
            }
        }
        return ne;
        
    }
};